localdev/seed/api.py

```diff
@@ -203,8 +203,38 @@ def _delete(self, model_cls, data):
         self.seed_result.add_deleted(deleted_type_counts)
         return deleted_count, deleted_type_counts
 
+    def _create_cms_resource_page(self, resource_page):
+        """ Create resource pages in CMS if they don't exists."""
+        existing_obj = ResourcePage.objects.filter(slug=resource_page["slug"]).first()
+        if existing_obj:
+            self.seed_result.add_existing(existing_obj)
+            return existing_obj
+
+        page_obj = ResourcePage(
+            title=resource_page["title"],
+            sub_heading=resource_page["sub_heading"],
+            content=json.dumps(resource_page["content"]),
+            slug=resource_page["slug"],
+        )
+        added_obj = get_top_level_wagtail_page().add_child(instance=page_obj)
+        self.seed_result.add_created(added_obj)
+        return added_obj
+
+    def _delete_cms_resource_page(self, resource_page):
+        """Delete resource pages from CMS."""
+        existing_obj = ResourcePage.objects.filter(slug=resource_page["slug"]).first()
+
+        if not existing_obj:
+            return 0, {}
+
+        deleted_count, deleted_type_counts = delete_wagtail_pages(
+            ResourcePage, {"slug": existing_obj.slug}
+        )
+        self.seed_result.add_deleted(deleted_type_counts)
+        return deleted_count, deleted_type_counts
```

Comment:
are we currently really using these returning vars `deleted_count, deleted_type_counts`  ?